src/assisted_test_infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -590,6 +590,17 @@ def set_ram_kib(self, node_name, ram_kib):
             raise Exception(f"Failed to set memory for node: {node_name}")
         log.info(f"Successfully set memory to {ram_kib} for node: {node_name}")
 
+    def set_ipxe_url(self, network_name, ipxe_url):
+        network = self.get_network_by_name(network_name)
+        current_xml = network.XMLDesc(0)
+        xml = minidom.parseString(current_xml.encode("utf-8"))
+        dhcpElement = xml.getElementsByTagName("dhcp")[0]
+        bootp = xml.createElement("bootp")
+        bootp.setAttribute("file", ipxe_url)
+        dhcpElement.appendChild(bootp)
+        self.destroy_network(network)
+        self.create_network(xml.toprettyxml())
```

Comment:
``network.update(...)`` doesn't work?

Reply:
I didn't realize that function existed! All the examples usually showed create and destroy. I found the function
```
    def update(self, command, section, parentIndex, xml, flags=0):
        """Update the definition of an existing network, either its live
        running state, its persistent configuration, or both. """
        ret = libvirtmod.virNetworkUpdate(self._o, command, section, parentIndex, xml, flags)
        if ret == -1:
            raise libvirtError('virNetworkUpdate() failed')
        return ret
```
Are you familiar with using it? I'm not sure what to pass in for the `command`, `section`, and `parentIndex` parameters 😅

Reply:
In fact I'm not so sure myself
Now seeing this function, if it's that undocumented and not trivial to use then we'd better off with the existing code